geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/AvroSimpleFeature.scala

```diff
@@ -56,7 +56,7 @@ class AvroSimpleFeature(id: FeatureId, sft: SimpleFeatureType)
   def setAttributesNoConvert(vals: Array[Object])= vals.zipWithIndex.foreach { case (v, idx) => values(idx) = v }
 
   def getAttributeCount = values.length
-  def getAttributes: JList[Object] = values.toList
+  def getAttributes: JList[Object] = java.util.Arrays.asList(values: _*)
```

Comment:
Any reason for this change?  (Just curious)

Reply:
@tkunicki pointed out a different piece of code in our spark module that was intended to prevent serialization issues with the result of `getAttributes` - i think this is the only place where it might not be serializable so i snuck in a fix